### DISCUSSION TITLE
Other websites

### DISCUSSION THREAD

**Discussion by Ryner95** [UltimaHoarder/UltimaScraper#1457] [Q&A]
How could I do to download from other pages of the same style, in this case I am looking to download from loverfans.com, to obtain the api how would it be done?

**Comment by OFfriend** [comment]
publisher = modelname

`https://api.loverfans.com/api/posts?visibility[]=open&visibility[]=premium,exclusive&visibility[]=followers&publisher=${publisher}&nextCursorToken=&limit=15`

look if the response has `metadata.hasMoreElements = true` than you can use `metadata.nextCursorToken`

 `https://api.loverfans.com/api/posts?visibility[]=open&visibility[]=premium,exclusive&visibility[]=followers&publisher=${publisher}&nextCursorToken=${res.body.metadata.nextCursorToken}&limit=15`
